renku/domain_model/workflow/plan.py

```diff
@@ -66,6 +67,7 @@ def __init__(
         self.description: Optional[str] = description
         self.id: str = id
         self.date_created: datetime = date_created or local_now()
+        self.date_modified: datetime = date_modified or local_now()
         self.invalidated_at: Optional[datetime] = invalidated_at
```

Comment:
This is really minor thing, so, don't bother with it if it's a headache: Should we rename `invalidate_at` to `date_removed` (like datasets)?

Reply:
Same for composite plans.